librecad/src/ui/forms/qg_commandwidget.cpp

```diff
@@ -155,7 +157,48 @@ void QG_CommandWidget::tabPressed() {
             leCommand->setText(reducedChoice.first());
         }
         else if (reducedChoice.count()>0) {
+            QString longestString = "";
+            QString shortestString = "";
```

Comment:
Ok, I see. An old habit from the times I learned C ('always initialize your variables', he said...).

I packed that code into a dedicated method that returns the longest common root on another branch. I would like to have your opinion on [this commit](https://github.com/Rupicapra-rupicapra/LibreCAD/commit/07e447b43b15c42954f3bccd2f776cf49c7b9468). If it seems OK to you, I can replace this pull request by one about these changes.
I put the method in the private part 'by default', because I don't think it's necessary to access it from the outside of the command-line, but maybe it's a mistake.